spec/controllers/straws_controller_spec.rb

```diff
@@ -2,63 +2,82 @@
 
 RSpec.describe StrawsController, :type => :controller do
   describe "POST /straws" do
-    it "returns a single username from the list of usernames" do
+    it "should return a single username from the list of usernames" do
       req = slack_request
       req[:text] = "@jess"
 
       post :create, req
 
       json_response = JSON.parse(response.body)
-      expect(json_response["text"]).to include("@jess")
+      expect(json_response["attachments"][0]["fields"][0]["value"]).to include("@jess")
     end
 
-    it "chooses a user name from the list of user names" do
+    it "should only return usernames with an '@' symbol" do
       req = slack_request
-      req[:text] = "@elizabeth @brent"
+      req[:text] = "jess"
 
       post :create, req
 
-      straws_response = JSON.parse(response.body)["text"]
+      json_response = JSON.parse(response.body)
+      expect(json_response["attachments"][0]["fields"][0]["value"]).to eq("")
```

Comment:
Nice. A _slightly_ better test would have been to include one with an @ and one without, and test that it returns only the @ username.